server/src/domain/domain.constant.ts

```diff
@@ -67,6 +67,33 @@ const validMimeTypes = [
   'image/x-sony-arw',
   'image/x-sony-sr2',
   'image/x-sony-srf',
+  'image/dng',
```

Comment:
This is a duplicate entry I believe.

Can we also sort this list alphabetically?

Reply:
Fixed on both counts.

Reply:
Am I missing something? Nothing changed?

Reply:
You're not missing something, I was. I forgot to push the updates from my local tree to github. Since Alex already pushed this to main, do I need to open a separate pull request to fix your concerns, or is there a way to re-open a pull request?

Reply:
You will need to open a follow-up PR. Thank you 🙂